setup.py

```diff
@@ -22,8 +22,14 @@
     packages=find_packages(exclude=['tests*']),
     include_package_data=True,
     zip_safe=False,
-    dependency_links=[],
-    install_requires=requirements,
+    dependency_links=[
+        ('git+https://www.github.com/hilearn/marshmallow-'
+         'annotations.git@a7a2dc96932430369bdef36555082df990ed9bef#egg=marshmallow-annotations')
+    ],
+    install_requires=[
```

Comment:
@allisonsuarez is there a reason we removed the requirements file from the `install_requires`? as it breaks the projects if we simply install metadata using setup.py, and hence via pip. Or it's just a typo? 

cc: @feng-tao